docs/src/cli/choose-a-cluster.md

```diff
@@ -0,0 +1,97 @@
+# Connecting to a Cluster
+
+Solana maintains several different clusters with different purposes.  Each
+cluster features a Solana-owned validator that serves as an entrypoint to the
+cluster.
+
+## Before you Begin
+
+ - Make sure you have first
+[installed the Solana command line tools](install-solana-cli-tools.md)
+
+## Choose a Cluster
+
+#### Mainnet Beta
+A permissionless, persistent cluster for early token holders and launch partners.
+No smart contracts or inflation.
+ * Tokens that are issued on Mainnet Beta are **real** SOL
+ * If you have paid money to purchase/be issued tokens, such as through our
+ CoinList auction, these tokens will be transferred on Mainnet Beta.
+   * Note: If you are using a non-command-line wallet such as
+   [Trust Wallet](../wallet/trust-wallet.md),
+   the wallet will always be connecting to Mainnet Beta.
+ * Entrypoint URL for Mainnet Beta: `https://api.mainnet-beta.solana.com`
+
+#### Devnet
+* Devnet serves as a playground for anyone who wants to take Solana for a
+test drive, as a user, token holder, app developer, or validator.
+* Application developers should target Devnet.
+* Potential validators should first target Devnet.
+* Key differences between Devnet and Mainnet Beta:
+  * Devnet tokens are **not real**
+  * Devnet includes a token faucet for airdrops for application testing
+  * Devnet may be subject to ledger resets
+  * Devnet typically runs a newer software version than Mainnet Beta
+  * Devnet may be maintained by different validators than Mainnet Beta
+ * Devnet is upgraded automatically on the last Monday of each month.
```

Comment:
IMO remove, this isn't too helpful and is not totally accurate.    " Devnet typically runs a newer software version than Mainnet Beta" is sufficient IMO
```suggestion

```